### DISCUSSION TITLE
Instagram - stuck

### DISCUSSION THREAD

**Discussion by Mas99** [AAndyProgram/SCrawler#120] [Q&A] [answered]
Hi,

I am trying to download some Instagram profiles but it is not working. I have checked the settings 100 times and they are ok. It doesn't show any error but just shows "Jobs 3"

I have created different account in case my instagram was banned etc. but still no luck.

Can somebody please help ?

**Comment by AAndyProgram** [comment, marked as answer]
1. If you're trying to download tagged posts it can take days or weeks. A few releases ago I notified users that they should not use tagged posts.
1. If the profile you are trying to download contains too many posts, please wait for a while (at least a few minutes). Instagram parser is quite slow.

If you keep seeing the text `Jobs 3`, it means that everything is fine with your credentials and SCrawler is collecting information.

Please pay attention to [this](https://github.com/AAndyProgram/SCrawler/wiki/Settings#instagram-limits) chapter. After 200 requests Instagram downloader becomes much slower.

Can I see a profile you are trying to download?

**Reply by Mas99** [reply]
Thank you. I am not downloading tagged posts. 

There are about 400 posts which I think it is normal. 

When you click "i" right below the progress bar, It shows "Instagram downloading 1/1 profiles data message and this has been here whole day.

**Reply by AAndyProgram** [reply]
I can't say more until I see the profile. But "a whole day" is too much to download 400 posts. This is impossible. Maybe that profile contains thousands of stories?! 😀

**Reply by Mas99** [reply]
Here is the profile

https://www.instagram.com/funadultchats/?fbclid=IwAR1MEs2G8DnScz9GtsynejY-RdKlDJ5qo7iksKHIr2meh0-q-qF3rayEj3Y

**Reply by AAndyProgram** [reply]
The profile is ok. If you've started and stopped downloading Instagram profiles multiple times, I think you've face the limit. What is the number of spent requests in the SCrawler Instagram settings do you see? I suggest you wait a few hours or a day and in the future download one big profile every few hours.

**Reply by Mas99** [reply]
it is showing 231 requests...

Can I just click on " download all " and then leave it download it whenever it starts/wants ?

**Reply by AAndyProgram** [reply]
It starts immediately. Of course you can.
Have you already started it? Everything is fine?

**Reply by Mas99** [reply]
I have started and left it . but when I came back, it has closed itself without completing any tasks

See image below of settings - Does this looks ok ?
![1](https://user-images.githubusercontent.com/34427111/232255033-bda03c84-4656-4887-859b-93ed006ef050.PNG)

**Reply by AAndyProgram** [reply]
A hash is not a URL. The hash is the value after the equal sign, 32 characters long. But the hash is only needed to download tagged data, so your settings are okay.

**Reply by AAndyProgram** [reply]
Is there data in the log?

**Reply by Mas99** [reply]
No, log is empty....
